APMrover2/sailboat.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "Rover.h"
+
+/*
+    Rover Sailboat functionality
+*/
+class Sailboat
+{
+public:
+
+    // var_info for holding Parameter information
+    static const struct AP_Param::GroupInfo var_info[];
+
+    // enabled
+    bool enabled() const { return enable != 0;}
```

Comment:
why not just returning enable ?

Reply:
this came up in the dev call in relation to bitmasks, it has to cast the int8 to a bool so i added != 0, but maybe its OK in this case? I think I will be adding more options to the enable so 2 or 3 would be valid values in the future rather than just 0 or 1.

Reply:
int8 to bool is easy ! a static_cast is also a solution, but if you intend to have more value in this flag then your soltution is correct